hudi-flink-datasource/hudi-flink/src/test/java/org/apache/hudi/sink/utils/ConsistentBucketStreamWriteFunctionWrapper.java

```diff
@@ -58,7 +58,9 @@ public void invoke(I record) throws Exception {
     HoodieFlinkInternalRow hoodieRecord = toHoodieFunction.map((RowData) record);
     ScalaCollector<HoodieFlinkInternalRow> collector = ScalaCollector.getInstance();
     assignFunction.processElement(hoodieRecord, null, collector);
-    writeFunction.processElement(collector.getVal(), null, null);
+    for (HoodieFlinkInternalRow row: collector.getVal()) {
```

Comment:
why change this line?

Reply:
`collector` is used to collect emitted records from `asisgnFunction`, it's possible `asisgnFunction` emits more than one records for one time processElement, e.g., for case global inedx, it may emits 1 DELETE + 1 INSERT.